source/user-manual/wazuh-dashboard/config-file.rst

```diff
@@ -724,6 +724,105 @@ Set the footer of the PDF reports. To use an empty footer, type a space " " in t
 | **Allowed values** | Any string                 |
 +--------------------+----------------------------+
 
+ISM rollover policy
+-------------------
+
+.. warning::
+
+   Modifying these options has no effect if you do it once the Wazuh dashboard started for the first time. Read about different methods to customize the rollover policy in :doc:`/user-manual/wazuh-indexer/index-life-management`.
+
+ism.rollover.enabled
+^^^^^^^^^^^^^^^^^^^^
+
+Toggles the verification of the rollover policy. If set to ``true``, the rollover policy will be applied to the indices that match the index patterns defined in ``ism.rollover.index_patterns``, only if there isn't any other policy managing these indices.
+
+This option does not enable or disable the policy itself. 
+
++--------------------+----------------------------+
+| **Default value**  | true                       |
++--------------------+----------------------------+
+| **Allowed values** | true, false                |
++--------------------+----------------------------+
+
+ism.rollover.index_patterns
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The policy will be applied to the indices that match the patterns defined here. The index format must match the pattern: ``^.*-\d+$``. The managed indices must use an alias. Set ``index.plugins.index_state_management.rollover_alias`` as the alias to rollover.
```

Comment:
Examples for the ``^.*-\d+$`` format seem to be needed here. Linking to index aliases in Wazuh docs is also required here.

Reply:
Extracted from [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/ilm-rollover.html#ilm-rollover-options). I think this pattern applies to every index and index pattern name, so maybe it would be better to explain it in the [indices section](https://documentation.wazuh.com/current/user-manual/wazuh-indexer/wazuh-indexer-indices.html).

I see there is a note there that says:

> The index name must not contain the characters #, \, /, *, ?, ", <, >, |, ,, and must not start with _, -, or +. Also, all the letters must be lowercase.

`^.*-\d+$` is mostly the same, but as a regular expression that translates to:

- `^`: regex start
- `.`: Any character except new line
- `*`: 0 or more characters
- `\d+`: 1 or more digits
- `$`: regex end

Examples: `logs-0001`, `wazuh-alerts-4x-2023-10-27`, `wazuh-alerts-010`, `my-custom-index-128589`

Reply:
If we say it's required to match the format ``^.*-\d+$`` we need to add an example of such condition, adding something like this: "for example ``wazuh-alerts-4x-2023-10-27``, ``my-custom-index-128589``"

Also, we mention aliases but we should be linking or explaining what aliases are.